tools/wasm/air-beautify-wasm/src/lib.rs

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 Fluence Labs Limited
+ * Copyright 2023 Fluence Labs Limited
```

Comment:
In AquaVM/Marine year in copyright, reflect the year when this file was first created (for historical reasons).

Reply:
this change is needed to trigger CHANGELOG generation. You can always just use git to learn when file was created. I can revert this and change something else if you think it's important

Reply:
The correct value is 2022 :)

Reply:
Looks so, ok, let's keep 2022